Azure.ResourceManager.Core/ResourceGroupContainer.cs

```diff
@@ -52,27 +71,35 @@ public async override Task<ArmResponse<ResourceGroup>> CreateAsync(string name,
                 g => new ResourceGroup(ClientOptions, new ResourceGroupData(g)));
         }
 
+        /// <inheritdoc/>
         public override ArmOperation<ResourceGroup> StartCreate(string name, ResourceGroupData resourceDetails, CancellationToken cancellationToken = default)
         {
             return new PhArmOperation<ResourceGroup, ResourceManager.Resources.Models.ResourceGroup>(
                 Operations.CreateOrUpdate(name, resourceDetails, cancellationToken),
                 g => new ResourceGroup(ClientOptions, new ResourceGroupData(g)));
         }
 
+        /// <inheritdoc/>
         public async override Task<ArmOperation<ResourceGroup>> StartCreateAsync(string name, ResourceGroupData resourceDetails, CancellationToken cancellationToken = default)
         {
             return new PhArmOperation<ResourceGroup, ResourceManager.Resources.Models.ResourceGroup>(
                 await Operations.CreateOrUpdateAsync(name, resourceDetails, cancellationToken).ConfigureAwait(false),
                 g => new ResourceGroup(ClientOptions, new ResourceGroupData(g)));
         }
 
+        /// <summary>
+        /// List the ResourceGroups.
+        /// </summary>
```

Comment:
missing param and return, summary doesn't match standards https://github.com/m-nash/azure-proto-sdk/blob/master/XmlDocStandard.md#list--listasync